style.css

```diff
@@ -1,5 +1,19 @@
 /* Write your CSS here! */
 
+* {
```

Comment:
We actually don't want to have a margin of zero and don't need to have the box-sizing set to border-box to accomplish what the prompt asked us